examples/vic-gov-au/test/e2e/integration/core-modules/landingPage/LandingPage.feature

```diff
@@ -107,10 +107,10 @@ Feature: Landing page
     # Image Gallery
     And the image gallery component should have the following items:
       | image                                                                                                                                          | title                                                |
-      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/2018-19%20State%20Budget.jpg                         | 1 / 4 - Demo: 2018-19 State Budget                   |
-      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Aerial%20shot%20of%20new%20housing%20development.jpg | 2 / 4 - Demo: Aerial shot of new housing development |
-      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Bendigo%20Hospital.jpg                               | 3 / 4 - Demo: Bendigo Hospital                       |
-      | https://nginx-php-content-vic-develop.lagoon.vicsdp.amazee.io/sites/default/files/2019-09/Melbourne%20tram.jpg                                 | 4 / 4 - Demo: Melbourne tram                         |
+      | 2018-19%20State%20Budget.jpg                         | 1 / 4 - Demo: 2018-19 State Budget                   |
```

Comment:
Drupal asset URLs include month they were uploaded, changed this to be a partial match